draft-ietf-tls-dtls13.md

```diff
@@ -486,6 +488,46 @@ protected portion.
 * Otherwise, the record MUST be rejected as if it had failed
 deprotection, as described in {{handling-invalid-records}}.
 
+{{demux}} shows this demultiplexing procedure graphically 
+taking DTLS 1.3 and earlier versions of DTLS into account. 
+~~~
+             +----------------+
+             | Decrypted      |
+             | Content Type   |
+             | (DCT)          |
+             |                |			 
+             |     DCT == 21 -+--> Alert
+       +---> |     DCT == 22 -+--> Handshake
+       |     |     DCT == 23 -+--> Application Data
+       |     |     DCT == 24 -+--> Heartbeat   
+       |     |     DCT == 26 -+--> ACK
+       |     |                |
+       |     +----------------+
+       |
+       |
+       |       Decryption
+       +-----------------------------------+
+                                           | 
+             +----------------+            |
+             | Outer Content  |            |
+             | Type (OCT)     |            |
+             |                |    /+------+-----+\
+packet  -->  | 31 < OCT < 64 -+--> DTLS Ciphertext
+             |                |    (header bits 
+             |                |     start with 001)
+             |                |			 
+             |   OCT == 20   -+--> ChangeCipherSpec (DTLS <1.3)
+             |   OCT == 21   -+--> (Plaintext) Alert
+             |   OCT == 22   -+--> (Plaintext) Handshake
+             |   OCT == 23   -+--> Application Data (DTLS <1.3)
+             |   OCT == 24   -+--> Heartbeat (DTLS <1.3)
+             |   OCT == 25   -+--> CID (DTLS 1.2)
+             |   OCT == 26   -+--> (Plaintext) ACK (DTLS 1.3)
+             +----------------+
```

Comment:
1. I would put this so it reads top to bottom
2. I would put the parentheticals after, as in "Alert (Plaintext)"

Reply:
Ack